### DISCUSSION TITLE
Build pimcore/pimcore php docker images for ARM

### DISCUSSION THREAD

**Discussion by x1oJ0** [pimcore/pimcore#10500] [Q&A / Support]
Hello, is there any chance to build your core php pimcore docker images for ARM architecture? Then we are able to use it better on Apple M1 chips without emulation. It shouldn't a big problem right? Just update the build process with something like `docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7,linux/arm/v8 ...`. Thank you.

**Comment by dpfaffenbauer** [comment]
Could you maybe provide a PR?

**Reply by x1oJ0** [reply]
Well I would like to but I've never worked with github actions.. I'm building images through shell script localy. But found this one https://github.com/marketplace/actions/docker-buildx#build-and-push-to-dockerhub that could help you?

**Reply by dpfaffenbauer** [reply]
it is basically a shell script.... https://github.com/pimcore/docker/blob/main/.github/workflows/build.yml#L42

**Reply by x1oJ0** [reply]
Hey there, is there any chance to check created PR if it's okey?

**Reply by dpfaffenbauer** [reply]
sorry, missed it. I approved it, will wait for brusch too

**Reply by x1oJ0** [reply]
Thanks, looking forward to it. 👍🏻
